src/mlpack/bindings/R/R_option.hpp

```diff
@@ -73,7 +73,7 @@ class ROption
     data.cppType = cppName;
 
     // Every parameter we'll get from R will have the correct type.
-    data.value = ANY(defaultValue);
+    data.value = MLPACK_ANY(defaultValue);
```

Comment:
I'm curious about c02eba2ec9a551be7a5a2cd536c282f33777dde7 ; were we possibly stepping on other "ANY"?

Reply:
@birm see https://github.com/mlpack/mlpack/pull/3233#discussion_r905670594. TLDR: macros with short names are an accident waiting to happen; using the MLPACK_ prefix considerably reduces likelihood of inadvertent clashes

Reply:
Thank you, and sorry I didn't think to read through the resolved comments.